src/radical/pilot/utils/prof_utils.py

```diff
@@ -199,6 +199,114 @@
+        'p_pmngr_submission'      : [ {ru.EVENT: 'submission_start' , ru.STATE: None                       },  
+                                      {ru.EVENT: 'submission_stop'  , ru.STATE: None                       }  ],
+        'p_pmngr_scheduling_init' : [ {ru.EVENT: 'submission_stop'  , ru.STATE: None                       },  
+                                      {ru.EVENT: 'state'            , ru.STATE: rps.PMGR_ACTIVE_PENDING    }  ], 
+        'p_pmngr_scheduling'      : [ {ru.EVENT: 'state'            , ru.STATE: rps.PMGR_ACTIVE_PENDING    },     # batch system queue time
+                                      {ru.EVENT: 'bootstrap_0_start', ru.STATE: None                       }  ],
+        'p_agent_ve_setup_init'   : [ {ru.EVENT: 'bootstrap_0_start', ru.STATE: None                       },  
+                                      {ru.EVENT: 've_setup_start'   , ru.STATE: None                       }  ], 
+        'p_agent_ve_setup'        : [ {ru.EVENT: 've_setup_start'   , ru.STATE: None                       },  
+                                      {ru.EVENT: 've_setup_stop'    , ru.STATE: None                       }  ], 
+        'p_agent_ve_activate_init': [ {ru.EVENT: 've_setup_stop'    , ru.STATE: None                       },  
+                                      {ru.EVENT: 've_activate_start', ru.STATE: None                       }  ], 
+        'p_agent_ve_activate'     : [ {ru.EVENT: 've_activate_start', ru.STATE: None                       },
+                                      {ru.EVENT: 've_activate_stop' , ru.STATE: None                       }  ],
+        'p_agent_install_init'    : [ {ru.EVENT: 've_activate_stop' , ru.STATE: None                       },
+                                      {ru.EVENT: 'rp_install_start' , ru.STATE: None                       }  ],
+        'p_agent_install'         : [ {ru.EVENT: 'rp_install_start' , ru.STATE: None                       },
+                                      {ru.EVENT: 'rp_install_stop'  , ru.STATE: None                       } ],
+        'p_agent_launching'       : [ {ru.EVENT: 'rp_install_stop'  , ru.STATE: None                       },
+                                      {ru.EVENT: 'state'            , ru.STATE: rps.PMGR_ACTIVE            }  ],
+        'p_agent_runtime'         : [ {ru.EVENT: 'state'            , ru.STATE: rps.PMGR_ACTIVE            },     # total pilot runtime
+                                     [{ru.EVENT: 'state'            , ru.STATE: rps.DONE                   },
+                                      {ru.EVENT: 'state'            , ru.STATE: rps.CANCELED               },
+                                      {ru.EVENT: 'state'            , ru.STATE: rps.FAILED                 } ]]
+    }
+}
+
+# Set of default unit durations for RADICAL-Analytics. All the durations
+# are contiguos.
+UNIT_DURATIONS_DEBUG = {
+    'consume' : {
```

Comment:
Similar to the command on `p_*`, I would expand `u_*` to `unit_*`, or better even to `task_*`.

Reply:
I am indeed concern about length. I think it is mnemonically simple to associate p and u to pilot and unit. I would try like this and see whether it is a major issue. Would you be fine with that?

Reply:
I don't like it, but also don't mind too much, no worries.